solutions/ac-MultipleOf2019.mdx

```diff
@@ -47,5 +47,41 @@ int main()
 ```
 
 </CPPSection>
+	
+<JavaSection>
+	
+```java
+import java.io.*;
+import java.util.*;
+
+public class MultipleOf2019 {
+	public static void main (String[] args) {
+		Kattio io = new Kattio();
+		String numberString = io.next();
+		int num = 0;
+		int stringLength = numberString.length();
+		int pow = 1;
+		int[] count = new int[2019];
+		count[0] = 1;
+
+		for (int i = stringLength - 1; i >= 0; i--) {
+			//find the remainder of the current number mod 2019
+			num = (num + pow * (numberString.charAt(i) - '0')) % 2019;
```

Comment:
maybe Character.getNumericValue would be more explicit?

Reply:
come on this is silver im sure all people can understand that subtracting '0' gets you the actual number

Reply:
no?

Reply:
Yeah I think the getNumericalValue method might be more understandable

Reply:
i feel like all of the courses like at alphastar include this idea of subracting characters in their intro to a programming language class